### PR TITLE
Fix ArgumentOutOfRangeException when getting PostedPlaybackInfo

### PR DESCRIPTION
**Pull request by DeeJayBro**
MediaInfoController.GetPostedPlaybackInfo method attempts to access the first item of the mediaInfo Array without checking if MediaInfoHelper.GetPlaybackInfo returned an ErrorCode

**Changes**
Skip Code that accesses mediaInfo MediaSources Array when an ErrorCode is returned

### CODE REVIEW

**Review by crobibero — COMMENTED** [review]
I’m not sure what the clients are expecting, but if we aren’t sending a successful playback info response I would expect a 400 status code 

@jellyfin/client

**Comment by DeeJayBro** [comment]
As far as i see it, the web doesn't handle the 400 return  correctly, but correctly displays an error message with "NoCompatibleStream" when my change is implemented

**Review by crobibero — APPROVED** [review]

File: Jellyfin.Api/Controllers/MediaInfoController.cs [hunk 1]
```diff
@@ -192,6 +192,11 @@ public async Task<ActionResult<PlaybackInfoResponse>> GetPlaybackInfo([FromRoute
                 _mediaInfoHelper.SortMediaSources(info, maxStreamingBitrate);
             }
 
+            if (info.ErrorCode != null)
```
Comment (Bond-009) [inline Jellyfin.Api/Controllers/MediaInfoController.cs:195]: Why here instead of line 163?

**Review by Bond-009 — CHANGES_REQUESTED** [review]

File: Jellyfin.Api/Controllers/MediaInfoController.cs (hunk 1 above)
Comment (DeeJayBro) [reply Jellyfin.Api/Controllers/MediaInfoController.cs:195]: Sounds good, will change!

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**